Packs/AccentureCTI/Integrations/ACTIVulnerabilityQuery/ACTIVulnerabilityQuery.py

```diff
@@ -45,8 +45,8 @@ def __init__(self, id, cvss3, cvss2, published, modified, description, dbot, rel
 
     def to_context(self):
         context_values = super().to_context()
-        context_values[self.CONTEXT_PATH]['CVSS2'] = self.cvss2
-        context_values[self.CONTEXT_PATH]['CVSS3'] = self.cvss3
+        context_values[self.CONTEXT_PATH]['CVSS']['CVSS2'] = self.cvss2
```

Comment:
this will break backward compatibility. We should avoid it.
You should update the yml outputs accordingly . Remove the existing outputs from the yml, add the new ones and add the new outputs in the code and not replace.

Reply:
@anara123 - It doesnt break backward compatibility as its only a change in context and it is still mapped the same into the indicator (we are changing the mapper as well).

Scripts that use this method of building an indicator (enrichers) are not affected as the attributes are not changed.

Reply:
It might affect playbooks though, instead of changing it I can leave it and add it in both places.

Reply:
Correct. Add it in both places.

Reply:
Already done :)